asa_parser/tests/test_parser.py

```diff
@@ -58,7 +58,22 @@ def test_show_failover_history(self):
         self.assertEqual(expected, asa.failover_history())
 
     def test_startup_config_errors(self):
-        self.assertEqual(True, True)
+        # create a new text file in the tests directory called show_startup_config_errors.txt
+        # this file will contain only the "show startup-config errors" section from the main
+        # showtech_primary.txt file.  This is done to separate testing functionality from the
+        # main production functionality.
+        asa = ap.AsaParser(os.path.join(self.txt_path, 'show_startup_config_errors.txt'))
+
+        # execute the parser and get the result
+        # for now this will simply be a list of all the text in the section
+        # the return is in JSON format
+        result = asa.startup_config_errors()
+
+        # make sure the text section appears in the JSON return
+        self.assertIn('"text":', result)
+
```

Comment:
Pull request looks good. Only question is if the "result" object was supposed to be returned as a JSON object or just in JSON format as a string. Looks to be the latter option, but the tests showed as passing according to the design decisions you made so everything is fine.

Reply:
Thanks - The intent was to return a JSON string - json.dumps() - for now.  So the unit tests simply handles the return as a string.  In sprint-3 we'll start actually parsing the data.  Likely will just use pandas.read_json() to parse the strings.